app/src/main/java/com/fluffybros/tujefe/ui/home/HomeFragment.kt

```diff
@@ -22,13 +22,29 @@ class HomeFragment : Fragment() {
         homeViewModel =
             ViewModelProvider(this).get(HomeViewModel::class.java)
         val root = inflater.inflate(R.layout.fragment_home, container, false)
-        val textView: TextView = root.findViewById(R.id.text_home)
-        homeViewModel.text.observe(
-            viewLifecycleOwner,
-            Observer {
-                textView.text = it
-            }
-        )
         return root
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val exampleList = generateHomeRecyclerItems()
+        recycler_home.apply {
+            layoutManager = LinearLayoutManager(activity)
+            adapter =
+                HomeRecyclerAdapter(exampleList)
+        }
+    }
+
+    private fun generateHomeRecyclerItems(): List<HomeRecyclerItem> {
+        val list = ArrayList<HomeRecyclerItem>()
```

Comment:
let's leave a comment here that this is a dummy list - make it a TODO , assigned to yourself